CMakeLists.txt

```diff
@@ -58,6 +58,11 @@ function(enableSanitizer SANITIZER)
     set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -fsanitize=${SANITIZER}" PARENT_SCOPE)
 endfunction()
 
+if(KVS_DEFAULT_STACK_SIZE)
+  message(STATUS "Building with default stack size")
```

Comment:
What happens here if the value is empty? We should have validation here. I see the validation in webrtc, but we should add it here too. 

Also, a nit: can we rename `KVS_DEFAULT_STACK_SIZE` to `KVS_DEFAULT_STACK_SIZE_BYTES` or something similar to be explicit about the unit.

Reply:
+1

Reply:
If the value is empty it's undefined and goes to 'else'

Reply:
renamed compile time variable to have _BYTES for readability.